app/timetables/tests/test_models.py

```diff
@@ -1,21 +1,40 @@
 from django.test import TestCase
 from django.db import IntegrityError
 
-from app.timetables.models import Weekday
+from app.timetables.models import Weekday, Meal
 
 
 class WeekdayTest(TestCase):
     """Tests the Weekday model."""
 
     def setUp(self):
-        Weekday.objects.create(name="monday")
+        Weekday.objects.create(name='monday')
 
     def test_weekday_name_should_be_capitalized_on_save(self):
-        day = Weekday.objects.get(name__iexact="monday")
+        day = Weekday.objects.get(name__iexact='monday')
 
-        self.assertEqual(day.name, "Monday")
+        self.assertEqual(day.name, 'Monday')
 
     def test_duplicate_weekday_name_cannot_be_saved(self):
-        day = Weekday(name="Monday")
+        day = Weekday(name='Monday')
 
         self.assertRaises(IntegrityError, day.save)
+
+
+class MealTest(TestCase):
+    """Tests the Meal model."""
+
+    def setUp(self):
+        Meal.objects.create(
+            name='breakfast', start_time='21:30:05', end_time='22:30:05'
```

Comment:
Since the start_time and end_time fields are time fields, do we want store time objects in them instead? 
```
from datetime import datetime
datetime.strptime('21:30:05', '%H:%M:%S').time()
```